src/main/java/school/lemon/changerequest/java/introduction/hw1/Task3.java

```diff
@@ -2,6 +2,6 @@
 
 public class Task3 {
     public static boolean isBetween(int a, int b, int c) {
-        return false;
+        return a <= b && c >= b || c <= b && b >= a || a > b && c < b;
```

Comment:
I'll simplify your expression and place term that you need to check on the first place (in our case we always need to check variable `b`):
```
return b>=a && b<=c || b>=c && b>=a || b<a && b>c;
```
From the task definition, we need to return true only if `b` is between `a` and `c`, so you only need to check 2 cases here: `b` is greater or equal to `a` **and** is smaller or equal to `c` **OR** `b` is greater or equal to `c` **and** is smaller or equal to `a`

Please rewrite your solution despite it passes all the test cases.